src/llm_utils.py

```diff
@@ -5,178 +5,136 @@
 from google import genai
 from typing import List, Dict
 
+from github_utils import get_valid_hunks
+
 def analyze_code_changes(diff_content: str) -> List[Dict]:
     """
     Analyze code changes using OpenAI's GPT model
     Returns a list of review comments
     """
-    API_KEY = os.getenv('GEMINI_API_KEY')
 
-    if not API_KEY:
-        raise ValueError("GEMINI_API_KEY is not set. Please add it to your environment variables.")
+    try:
 
-    # Prepare the prompt for the LLM
-    prompt = f"""
-    Analyze the following code changes and provide review comments using EXACTLY this format:
+        API_KEY = os.getenv('GEMINI_API_KEY')
```

Comment:
Error handling for missing API key
```suggestion
|
```
N/A